content/applications/sales/sales/invoicing/invoicing_policy.rst

```diff
@@ -4,78 +4,83 @@ Invoice based on delivered or ordered quantities
+   the :guilabel:`Automatic invoice` feature, which automatically generates invoices when an online
+   payment is confirmed.
 
-Choose an invoicing policy on a product form
-============================================
+Invoicing policy on product form
+================================
 
-From any product page, under the **Sales tab**, you will find the invoicing policy, which can be
-manually changed.
+On any product page (:menuselection:`Sales app --> Products --> Products --> Choose any desired
+product), the :guilabel:`Invoicing Policy` option is located under the :guilabel:`General
+Information` tab, where it can be changed manually.
 
-.. image:: invoicing_policy/invoicing_policy_5.png
+.. image:: invoicing_policy/invoicing-policy-general-info-tab.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to change your invoicing policy on a product form on Odoo Sales?
+   :alt: How to change your invoicing policy on a product form on Odoo Sales.
 
 Impact on sales flow
 ====================
 
-On Odoo Sales, the basic sales flow will be to create a quotation, send it to your customer,
-wait for confirmation, confirm the sales order and create an invoice.
+In Odoo *Sales*, the basic sales flow starts with the creation of a quotation. Then, that quotation
+is sent a customer. Next, it needs to be confirmed, which turns the quotation into a sales order.
+This, in turn, creates an invoice.
 
-- **Invoice what is ordered**: No impact on this basic sales flow. Indeed, you can invoice as soon
-  as the sale is confirmed.
+The following is a breakdown of how invoicing policy rules impact the aforementioned sales flow:
 
-- **Invoice what is delivered**: Small impact on sales flow because you will have to manually
-  enter the delivered quantity on the sales order or to install the **Inventory App** to confirm the
-  delivered quantity before creating an invoice, with the **Sales App**. Indeed, if you try to
-  create an invoice without validating the delivered quantity, you will receive an error message as
-  below.
+- :guilabel:`Invoice what is ordered`: No impact on the basic sales flow. An invoice is created as
+  soon as a sale is confirmed.
 
-  .. image:: invoicing_policy/invoicing_policy_3.png
-     :class: img-thumbnail
-     :alt: How the choice of your invoicing policy impacts your sales flow on Odoo Sales?
+- :guilabel:`Invoice what is delivered`: Minor impact on sales flow, because the delivered quantity
+  needs to be manually entered on the sales order. Or, the *Inventory* app can be installed and used
+  to confirm the delivered quantity before creating an invoice with the *Sales* app.
+
+.. warning::
+   If a user attempts to create an invoice without validating the delivered quantity, the following
+   error message appears: "There is no invoiceable line. If a product has a Delivered quantities
+   invoicing policy, please make sure that a quantity has been delivered."
```

Comment:
This is text that appears in the UI, so it deserves a guilabel. I think you could include the double quotes if you wanted too 🙂
```suggestion
   error message appears: :guilabel:`There is no invoiceable line. If a product has a Delivered
   quantities invoicing policy, please make sure that a quantity has been delivered.`.
```